config/example-extras.cfg

```diff
@@ -1221,9 +1221,14 @@
 #a0_pin:
 #   The pins connected to an uc1701 type lcd. These parameters must be
 #   provided when using an uc1701 display.
-#contrast: 40
+#rs_pin:
+#cs_pin:
+#a0_pin:
+#   The pins connected to an st7567 type lcd. These parameters must be
+#   provided when using an st7567 display.
+#contrast: 50
 #   The contrast to set when using a uc1701 type display.  The value may
-#   range from 0 to 63.  Default is 40.
+#   range from 0 to 63.  Default is 50.
```

Comment:
It is not possible to see any text with default contrast 40 and can be misleading during configuration.

Reply:
It depends on the display.  The display that I developed UC1701 support with shows well at 40, not great anywhere else.  If ST7567 needs a different default value then I recommend making it different.  By making this change you will affect working UC1701 configurations.

Reply:
If UC1701 shows well at 40 then I agree with different default values per display type. I made it different for ST7567.